pkg/dispatcher/cache/resource_binding_processor.go

```diff
@@ -93,11 +93,47 @@ func (dc *DispatcherCache) unSuspendResourceBinding(rb *workv1alpha2.ResourceBin
 }
 
 func (dc *DispatcherCache) patchUnSuspendResourceBinding(rb *workv1alpha2.ResourceBinding) error {
-	patchBytes, err := json.Marshal([]jsonpatch.Operation{
-		{Operation: "add", Path: "/spec/suspension", Value: map[string]interface{}{
-			"scheduling": false,
-		}},
-	})
+	dispatchContent := &api.DispatchContent{
+		ResourceRequest: rb.Spec.ReplicaRequirements.ResourceRequest,
+	}
+	for _, cluster := range rb.Spec.Clusters {
+		dispatchContent.Replicas += cluster.Replicas
+	}
+	content, err := json.Marshal(dispatchContent)
+	if err != nil {
+		return err
+	}
+	newRb := rb.DeepCopy()
+	if newRb.Spec.Suspension != nil {
+		newRb.Spec.Suspension.Dispatching = ptr.To(false)
```

Comment:
Dispatching or Scheduling? The old code is trying to update the Scheduling field